app/services/sufia/user_stat_importer.rb

```diff
@@ -5,7 +5,11 @@ class UserStatImporter
     UserRecord = Struct.new("UserRecord", :id, :user_key, :last_stats_update)
 
     def initialize(options = {})
-      @verbose = options[:verbose]
+      if options[:verbose]
```

Comment:
This is only if they specify `verbose`.  I'm using the pattern that is in Rails itself:  https://github.com/rails/rails/blob/a26a3a075637215c9028308436ca89cba8da2ed5/railties/lib/rails/commands/server.rb#L123-L133